examples/axios.test.js

```diff
@@ -984,4 +984,13 @@ describe('axios', function() {
     });
     // acquit:ignore:end
   });
+  it('user-agent-test', async function() {
+    await axios.get('https://httpbin.org/get', {
+      headers:
+      {'User-Agent': 
+      'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246'} } )
```

Comment:
Let's do a simpler user-agent string, like "Axios v2.x" or whatever the current Axios version is